### PR TITLE
style: fix linting issues

### PR DESCRIPTION
**Pull request by SimonThormeyer**
# What's new in this PR
Some CI actions were failing due to linting errors or missing formatting. This PR fixes those issues.

----
##### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: crypto/src/mls/buffer_external_commit.rs [hunk 1]
```diff
@@ -30,7 +30,7 @@ impl MlsCentral {
 }
 
 #[cfg(test)]
-pub mod tests {
```
Comment (istankovic) [inline crypto/src/mls/buffer_external_commit.rs:33]: I think we should revisit all places like this that shouldn't be `pub` and just make them private.

Comment (augustocdias) [reply crypto/src/mls/buffer_external_commit.rs:33]: although they don't need to be public, it doesn't really matter since this is only for test targets

Comment (SimonThormeyer) [reply crypto/src/mls/buffer_external_commit.rs:33]: It's a style issue I guess... I changed it to private

File: crypto/src/mls/client/identities.rs [hunk 2]
```diff
@@ -206,7 +206,7 @@ pub mod tests {
         }
     }
 
-    pub mod push {
+    pub(crate) mod push {
```
Comment (istankovic) [inline crypto/src/mls/client/identities.rs:209]: Why do we need this if the parent mod is not pub? Similarly for `find` above.

Comment (SimonThormeyer) [reply crypto/src/mls/client/identities.rs:209]: Suprised that cargo check didn't catch this. Updating this in a minute.

File: crypto/src/mls/conversation/commit.rs [hunk 3]
```diff
@@ -374,7 +374,7 @@ pub mod tests {
 
     wasm_bindgen_test_configure!(run_in_browser);
 
-    pub mod add_members {
+    pub(crate) mod add_members {
```
Comment (istankovic) [inline crypto/src/mls/conversation/commit.rs:377]: This should also be private. Likewise for the rest of the modules in `tests`.

File: interop/src/build/web/wasm.rs [hunk 4]
```diff
@@ -19,7 +19,7 @@ use crate::TEST_SERVER_PORT;
 use color_eyre::eyre::Result;
 use std::net::SocketAddr;
 
-pub async fn find_wasm_file(deploy_path: &std::path::Path) -> Result<std::path::PathBuf> {
+pub(crate) async fn find_wasm_file(deploy_path: &std::path::Path) -> Result<std::path::PathBuf> {
```
Comment (istankovic) [inline interop/src/build/web/wasm.rs:22]: Why does this need to be `pub(crate)`? I don't see it used outside of this module.

**Review by istankovic — APPROVED** [review]

**Pull request merged**